src/persist-client/src/impl/machine.rs

```diff
@@ -279,6 +283,7 @@ where
     >(
         &mut self,
         cmd: &CmdMetrics,
+        metrics: &Metrics,
```

Comment:
curious - is there a reason this needs to be passed in as a param vs accessing the metrics via `self`?

Reply:
🤦 it's been a week, apparently. done!